src/commands/lambda/functions/uninstrument.ts

```diff
@@ -70,6 +71,45 @@ export const getFunctionConfig = async (
   }
 }
 
+export const getFunctionConfigsFromRegEx = async (
```

Comment:
it looks like this function is very similar to : https://github.com/DataDog/datadog-ci/blob/e56249423b2b6bf637b66aea5e36471b3d50e047/src/commands/lambda/functions/instrument.ts#L91
maybe this could be refactored?

Reply:
Just refactored the function, there was some code that could be used by both instrument/uninstrument, even though I reduced the use of repetitive code, there might be still some of it, please let me know what you think of the modification